tests-hivemq-platform-operator/build.gradle.kts

```diff
@@ -81,7 +81,7 @@ val integrationTest by tasks.registering(Test::class) {
         val testPlan = environment["TEST_PLAN"].toString()
         if (testPlan == "Other") {
             systemProperty(
-                "excludeTags", "Upgrade,Extensions,Services1,Services2,CustomConfig,Services,Platform,PodSecurityContext"
+                "excludeTags", "Upgrade,Extensions,CustomConfig,Containers,Services,Platform,PodSecurityContext"
```

Comment:
In the GitHub action I still see `Services1` and `Services2` being used. Why is this just `Services` in here? I assume all `Services1` and `Services2` tests have also the `Services` tag, so this just saves to list all sub-tags? Nice. I think I'll copy this in the platform operator with all the extension tests.

Reply:
Nice, I've already copied this over for the operator.